pyproject.toml

```
@@ -0,0 +1,113 @@
+[build-system]
+requires = ["hatchling"]
+build-backend = "hatchling.build"
+
+[project]
+name = "EnergyFlow"
+dynamic = ["version"]
+description = "Python package for the Energy Flow suite of particle physics tools"
+readme = "README.md"
+requires-python = ">= 3.7"
+authors = [
+    { name = "Patrick T. Komiske III", email = "pkomiske@mit.edu" },
+]
+keywords = [
+    "CMS",
+    "Deep Sets",
+    "EFM",
+    "EFN",
+    "EFP",
+    "EMD",
+    "Earth Mover Distance",
+    "Energy Flow Moment",
+    "Energy Flow Network",
+    "Energy Flow Polynomial",
+    "MOD",
+    "Open Data",
+    "PFN",
+    "Particle Flow Network",
+    "Wasserstein",
+    "architecture",
+    "collider",
+    "correlator",
+    "energy flow",
+    "energyflow",
+    "jets",
+    "metric",
+    "multigraph",
+    "neural network",
+    "open",
+    "physics",
+    "polynomial",
+    "substructure",
+]
+classifiers = [
+    "Development Status :: 5 - Production/Stable",
+    "Intended Audience :: Developers",
+    "Intended Audience :: Science/Research",
+    "License :: OSI Approved :: GNU General Public License v3 (GPLv3)",
+    "Natural Language :: English",
+    "Operating System :: MacOS",
+    "Operating System :: Microsoft :: Windows",
+    "Operating System :: POSIX :: Linux",
+    "Operating System :: Unix",
+    "Programming Language :: C++",
+    "Programming Language :: Python",
+    "Programming Language :: Python :: 3",
+    "Programming Language :: Python :: 3 :: Only",
+    "Programming Language :: Python :: 3.7",
+    "Programming Language :: Python :: 3.8",
+    "Programming Language :: Python :: 3.9",
+    "Programming Language :: Python :: 3.10",
+    "Topic :: Scientific/Engineering",
+    "Topic :: Scientific/Engineering :: Information Analysis",
+    "Topic :: Scientific/Engineering :: Mathematics",
+    "Topic :: Scientific/Engineering :: Physics",
+    "Topic :: Software Development :: Libraries :: Python Modules",
+]
+dependencies = [
+    "h5py >= 2.9.0",
+    "importlib-metadata>=1.4.0; python_version<\"3.8\"",
+    "numpy >= 1.12.0",
+    "six",
+    "wasserstein >= 1.0.1",
+]
+
+[project.optional-dependencies]
+all = [
+    "igraph",
+    "scikit-learn",
+    "tensorflow >= 2.5.0",
+]
+archs = [
+    "scikit-learn",
+    "tensorflow >= 2.5.0",
+]
+examples = [
+    "matplotlib",
+    "scikit-learn",
+    "tensorflow >= 2.5.0",
+]
+generation = [
+    "igraph",
+]
+tests = [
+    "igraph",
+    "pot >= 0.8.0",
+    "pytest",
+    "scikit-learn",
+    "tensorflow >= 2.5.0",
+]
+
+[project.urls]
+Homepage = "https://energyflow.network"
+Issues = "https://github.com/pkomiske/EnergyFlow/issues"
+"Source Code" = "https://github.com/pkomiske/EnergyFlow"
```

```suggestion
Issues = "https://github.com/thaler-lab/EnergyFlow/issues"
"Source Code" = "https://github.com/thaler-lab/EnergyFlow"
```

Just updating the metadata for the new URL

---

Great catch. Thank you!